### PR TITLE
PR for 2.2.4 bugfix release

### PR DESCRIPTION
**Pull request by apeltzer**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/smrnaseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/smrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/nf-core/custom/dumpsoftwareversions/environment.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::multiqc=1.15
```
Comment (maxulysse) [inline modules/nf-core/custom/dumpsoftwareversions/environment.yml:6]: there's a newer version of this module

**Comment by maxulysse** [comment]
no multiqc update?

File: nextflow.config [hunk 2]
```diff
@@ -272,7 +287,7 @@ manifest {
     description     = """Small RNA-Seq Best Practice Analysis Pipeline."""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version = '2.2.3'
+    version = '2.2.4'
     doi             = ''
```
Comment (FriederikeHanssen) [inline nextflow.config:291]: no doi?

**Review by FriederikeHanssen — APPROVED** [review]
lgtm

**Review by maxulysse — APPROVED** [review]

**Comment by apeltzer** [comment]
> no multiqc update?

modules/nf-core/multiqc/environment.yml ? 😉

File: nextflow.config (hunk 2 above)
Comment (apeltzer) [reply nextflow.config:291]: Good one will add that 👍😅

**Pull request merged**